### PR TITLE
Initial Release of Humanitec orb [semver:major]

### PR DESCRIPTION
**Pull request by christophcrichter**
Initial release of Humanitec orb allowing for a simple integration between CircleCI and Humanitec.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,164 +1,9 @@
...
-      greeting:
-        default: Hello
-        description: Select a proper greeting
-        type: string
-    steps:
-    - run:
-        command: echo << parameters.greeting >> world
-        name: Hello World
-description: |
-  Sample orb description # What will your orb allow users to do? # Descriptions should be short, simple, and clear.
```
Comment (nickbenCCI) [inline README.md:119]: Suggested Orb Description: Build and push new images to Humanitec and improve maintenance of your Kubernetes-ready applications after every CircleCI build. Experience seamless continuous delivery during your development process.

Comment (christophcrichter) [reply README.md:119]: Thanks for the proposal. I will adjust it.

File: src/@orb.yml [hunk 2]
```diff
@@ -1,14 +1,9 @@
 version: 2.1
 
 description: >
-  Sample orb description
-  # What will your orb allow users to accomplish?
-  # Descriptions should be short, simple, and informative.
+  Easily integrate CircleCI with Humanitec to achieve Continuous Delivery.
+  Build and push new images to Humanitec. Notify Humanitec about every new build.
```
[the remark this answers is not in the document]
Comment (christophcrichter) [reply src/@orb.yml:5]: Thanks for the proposal. I will adjust it.

**Comment by KyleTryon** [comment]
@christophcrichter Looks good to me 👍 . There is a small linting error fix needed to make the test pass. Once that is. green, I would recommend a "squash commit", and ensure "[semver:major]" remains in the commit message subject.

**Pull request merged**